webrtc-stats.html

```diff
@@ -118,9 +113,56 @@
         Basic concepts
       </h2>
       <p>
-        The stats API is defined in [[!WEBRTC]]. It is defined to return a set of objects that are
-        a subclass of the RTCStats dictionary. This is normatively defined in [[!WEBRTC]], but is
-        reproduced here for ease of reference.
+        The basic object of the stats model is the <a>stats object</a>. The following terms are
+        defined to describe it:
+      </p>
+      <dl>
+        <dt>
+          <dfn>Stats object</dfn>
+        </dt>
+        <dd>
+          <p>
+            This represents an internal object that keeps a set of data values, together with a
```

Comment:
I think we can and should define this API without specifying an internal object that's not exposed anywhere, that may or may not exist (it doesn't in Firefox). Even if we can't, I find the webrtc terminology more intuitive (from [id](http://w3c.github.io/webrtc-pc/#dom-rtcstats-id)):

*"A unique id that is associated with the object that was inspected to produce this RTCStats object. Two RTCStats objects, extracted from two different RTCStatsReport objects, MUST have the same id if they were produced by inspecting the same underlying object. User agents are free to pick any format for the id as long as it meets the requirements above."*

I think most people when they hear "stats object" will think of the RTCStats object, not the "underlying object" which is only mentioned indirectly. I think we should define it that way, otherwise we risk confusing people.